src/util.h

```diff
@@ -178,77 +178,90 @@ inline TypeName* Unwrap(v8::Local<v8::Object> object);
 
 inline void SwapBytes(uint16_t* dst, const uint16_t* src, size_t buflen);
 
-class Utf8Value {
+// Allocates an array of member type T. For up to StackStorageSize items,
+// the stack is used, otherwise malloc().
+template <typename T, size_t kStackStorageSize>
```

Comment:
Suggestion: `size_t kStackStorageSize = 1024` so you don't have to repeat when inheriting/instantiating.